code/game/machinery/defibrillator_mount.dm

```diff
@@ -159,12 +167,11 @@
 	wallframe_type = /obj/item/wallframe/defib_mount/charging
 
 /obj/machinery/defibrillator_mount/charging/process()
-	if(defib?.cell?.charge < defib.cell.maxcharge && is_operational())
+	if(defib.cell.charge < defib.cell.maxcharge && is_operational())
```

Comment:
why not use get_cell()

Reply:
var/obj/item/cell/C = get_cell()
if(C.charge < C.maxcharge && is_operation())

etc

Reply:
Changes should be made now.